### PR TITLE
fix: tablet sidebar profile being too wide and not centered 

### PR DESCRIPTION
**Pull request by rtpa25**
## What does this PR do?

fixed the sidebar being too wide and not centred on tablet <md type screen>

Fixes # (issue)

- https://github.com/calcom/cal.com/issues/6484

**Environment**: Staging(main branch) / Production

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: packages/prisma/.env [hunk 1]
```diff
@@ -1 +1 @@
-../../.env
```
Comment (sean-brydon) [inline packages/prisma/.env:1]: Can we uncommit this from PR please as it isnt needed for this fix :)

**Review by sean-brydon — DISMISSED** [review]
LGTM - just awaiting the .env file to be removed from this PR.

(Can't do this myself due to it being on a fork)

**Comment by rtpa25** [comment]
Will do it today in a couple of hours

**Review by PeerRich — APPROVED** [review]
tested and works! waiting for .env to be removed

**Pull request merged**